src/hiero_sdk_python/executable.py

```
@@ -261,33 +298,72 @@ def _execute(self, client: "Client"):
+            err_persistant,
+        )
+
+        raise MaxAttemptsError(
+            "Exceeded maximum attempts for request",
+            self.node_account_id,
+            err_persistant,
+        )
 
-def _delay_for_attempt(request_id: str, current_backoff: int, attempt: int, logger, error):
+
+def _delay_for_attempt(
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_delay_for_attempt`

Add return type annotation: `None`

(ANN202)